test/google_search_data_viewer/keywords/keyword_test.exs

```diff
@@ -4,11 +4,40 @@ defmodule GoogleSearchDataViewer.Keywords.KeywordTest do
   alias GoogleSearchDataViewer.Keywords.Keyword
 
   describe "create_keyword_uploads/2" do
-    test "give a valid list of keywords and a user, creates keywords for the user" do
+    test "given a valid list of keywords and a user, creates keywords for the user" do
       user = insert(:user)
       keywords = ["dog", "cat", "fish"]
 
-      assert Keyword.create_keyword_uploads(keywords, user) == Enum.count(keywords)
+      {keyword_count, _keywords} = Keyword.create_keyword_uploads(keywords, user)
+
+      assert keyword_count == Enum.count(keywords)
+    end
+  end
+
+  describe "get_keyword_uploads_for_user/1" do
+    test "given an existing user with uploaded keywords, lists keywords for the user" do
+      user = insert(:user)
+
+      keywords = ["dog", "cat", "fish"]
+
+      keyword_uploads =
+        Enum.map(keywords, fn keyword -> insert(:keyword_upload, name: keyword, user: user) end)
+
+      assert keyword_uploads ==
+               user
+               |> Keyword.get_keyword_uploads_for_user()
+               |> Repo.preload(:user)
```

Comment:
Should we simplify this a bit?

```suggestion
Keyword.get_keyword_uploads_for_user(user) == keywords
```

Reply:
I need to `preload` the User field for each KeywordUpload returned from `get_keyword_uploads_for_user` because the User field is lazily loaded, ie a join is not used to fetch the User for each KeywordUpload. 

If I used pattern matching instead of `==` I could match just the `name` field rather than all fields in the KeywordUpload struct. 

Let me know what you think.